src/xcode/ENA/ENA/Source/Services/TraceWarningDownload/__tests__/TraceWarningPackageDownloadTests.swift

```diff
@@ -93,7 +93,7 @@ class TraceWarningPackageDownloadTests: XCTestCase {
 		
 		// THEN
 		waitForExpectations(timeout: .medium)
-		XCTAssertEqual(responseCode, .noCheckins)
+		XCTAssertEqual(responseCode, .success)
```

Comment:
Because we have these special success cases just for testing and nothing else, I recommend to use this case. 
And because we have no other test to proof for a .noCheckin success, I would think that the code does not what the test expect...Could you check that again?

Reply:
I added a new case for this